.github/workflows/compile.yml

```diff
@@ -0,0 +1,38 @@
+name: compile
+on: [push, workflow_dispatch]
+
+jobs:
+  job:
+    name: ${{ matrix.os }}-${{ github.workflow }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: windows-latest
+            triplet: x64-windows
+          - os: ubuntu-latest
+            triplet: x64-linux
+          - os: macos-latest
+            triplet: x64-osx
+    steps:
+      - uses: actions/checkout@v3
+      # Install CMake
+      - uses: lukka/get-cmake@v3.24.2  # CMake 3.24.2, Ninja 1.11.1
+      # Install NASM
+      - uses: ilammy/setup-nasm@v1
+      # Launch the MSVC Tools Command Prompt (Windows)
+      - uses: ilammy/msvc-dev-cmd@v1
+      # Initialize the CMake directory and build
+      - name: Build
+        run: |
+          cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_NO_SUBFOLDERS=1
+          cmake --build build --config Release
+      # Upload the compiled binary
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.triplet }}
+          path: |
+            build/jpegoptim
```

Comment:
Just double checking, shouldn't we include LICENSE etc in the archives? https://github.com/XhmikosR/jpegoptim-windows/blob/521ec66948f69a677608f85d03742b4100397bda/.github/workflows/ci.yml#L40-L44

Reply:
Good point. I added those to the archive, and added the necessary acknowledgement in the README for distributing statically-linked libjpeg distributions from clause 2 of the IJG license under which standard libjpeg and libjpeg-turbo/MozJPEG are licensed.